src/client.js

```diff
@@ -51,35 +51,40 @@ export default class DockerExecWebsocketClient extends EventEmitter {
     //set state, state does nothing yet
     this.state = msgcode.pause;
 
-    this.stdin = through((data) => {
+    this.stdin = through2((data, enc, cb) => {
```

Comment:
I'm not really sure what omitting cb does, but I tried removing them and a test started failing, so I just left them in because it doesn't seem to do any harm to leave the callbacks in.